src/app/public/modules/tokens/tokens.component.ts

```diff
@@ -208,6 +276,9 @@ export class SkyTokensComponent implements OnInit, OnChanges, OnDestroy {
     this.notifyTokenSelected(token);
   }
 
+  /**
+   * @internal
```

Comment:
☝️ The docs tools only exposes `Input` and `Output`; we don't need all these `@internal` comments.

Reply:
Removed.